examples/iam-assumable-roles-with-saml/main.tf

```diff
@@ -0,0 +1,26 @@
+provider "aws" {
+  region = "eu-west-1"
+}
+
+resource "aws_iam_saml_provider" "idp_saml" {
+  name                   = "idp_saml"
+  saml_metadata_document = "${file("${path.module}/saml-metadata.xml")}"
```

Comment:
Could you please add this file also? We need to make sure that examples are always working with just `terraform apply`.

Reply:
Fixed, tested.